test/include/meshgenerators/StitchedSubgenerators.h

```diff
@@ -0,0 +1,33 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "StitchedMeshGenerator.h"
+
+// Forward declarations
+class StitchedSubgenerators;
+
+template <>
+InputParameters validParams<StitchedSubgenerators>();
+
```

Comment:
```suggestion
```

Remove all of this - the forward declaration is needed for the legacy param syntax, which we don't support for new objects